pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/offline/TableDataManagerProvider.java

```diff
@@ -15,54 +15,49 @@
  */
 package com.linkedin.pinot.core.data.manager.offline;
 
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
 import com.google.common.base.Preconditions;
 import com.linkedin.pinot.common.metrics.ServerMetrics;
 import com.linkedin.pinot.core.data.manager.config.TableDataManagerConfig;
 import com.linkedin.pinot.core.data.manager.realtime.RealtimeTableDataManager;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 
 /**
  * Provide static function to get PartitionDataManager implementation.
  *
  */
 public class TableDataManagerProvider {
-  private static ServerMetrics SERVER_METRICS;
-
-  private static Map<String, Class<? extends TableDataManager>> keyToFunction =
-      new ConcurrentHashMap<String, Class<? extends TableDataManager>>();
-
-  static {
-    keyToFunction.put("offline", OfflineTableDataManager.class);
-    keyToFunction.put("realtime", RealtimeTableDataManager.class);
+  private TableDataManagerProvider() {
```

Comment:
This is a util class, which is not allowed to instantiate (The method is defined private to prevent instantiate).